README.markdown

```diff
@@ -31,7 +31,8 @@ This should give you the flexibility you need for customizing your input/output
 
 ### Building Your Phar
 
-Simply run `php package.php`, which will create the file: `build/doctrine-migrations.phar` for you.
+The PHAR is ready for [download here](http://github.com/downloads/ericclemmons/migrations/doctrine-migrations.phar), but if you still would like to build your PHAR
```

Comment:
@TomasVotruba Indeed I would always focus on stable versions and if anyone uses alphas, overnight builds etc let them generated their own PHAR. I am a bit new to doctrine but I might create a pull request if I work out how to do these PHAR builds through travis. A *after_success* hook/step looks like the right place on travis.